stock_move_manual_lot/models/stock_move_line.py

```diff
@@ -46,5 +52,15 @@ def _reserve_manual_lot(self, vals):
                     owner_id=this.owner_id,
                     ml_to_ignore=self,
                 )
+                self.env.cr.execute(
+                    'select id from stock_picking where write_date > %s', (now,),
+                )
+                updated_picking_ids = list(map(
+                    operator.itemgetter(0), self.env.cr.fetchall(),
+                ))
 
             this.lot_id = this.manual_lot_id
+
+            if updated_picking_ids:
+                # see if we can assign pickings that were unassigned above
+                self.env['stock.picking'].browse(updated_picking_ids).action_assign()
```

Comment:
I think `action_assign` might raise, so you might want to wrap in try + saveoint.

Reply:
thanks, done

Reply:
Thanks! You may squash now.